src/Microsoft.ServiceFabric.Actors/Runtime/ActorConcurrencyLock.cs

```diff
@@ -93,37 +93,37 @@ internal int Test_CurrentCount
             await this.reentrantLock.WaitAsync(cancellationToken);
             try
             {
-                // A new logical call context is appended to every outgoing method call.
+                // Specifies a new logical call context is appended to every outgoing method call.
                 // The received callContext is of form callContext1callContext2callContext3... 
                 // thus to check if incoming call was made from the current calls in progress
-                // we need to check if the incoming call context starts with the currentCallContext
+                // we need to check if the incoming call context starts with the currentCallContext.
                 var startsWith = incomingCallContext.StartsWith(this.currentCallContext);
 
                 if (startsWith)
                 {
-                    // the incoming call is part of the current call chain
+                    // Indicates the incoming call is a part of the current call chain.
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.